examples/segmentation_3d/unet_evaluation_dict.py

```diff
@@ -52,7 +51,7 @@
     Rescaled(keys=['img', 'seg']),
     ToTensord(keys=['img', 'seg'])
 ])
-val_ds = monai.data.Dataset(data=val_files, transform=val_transforms)
```

Comment:
Why only change in this example?
How about other examples using MONAI Dataset?
Thanks.

Reply:
fixed in my latest patch.
Thanks.